### PR TITLE
[Blazor] Move all test projects to Generic host

### PR DESCRIPTION
**Pull request by javiercn**
* Moves all test projects to generic host and cleans up adapters

### CODE REVIEW

File: src/Components/Blazor/testassets/HostedInAspNet.Server/Program.cs [hunk 1]
```diff
@@ -14,12 +13,12 @@ public static void Main(string[] args)
...
-                    .AddCommandLine(args)
-                    .Build())
-                .UseStartup<Startup>()
-                .Build();
+        public static IHost BuildWebHost(string[] args) =>
+            Host.CreateDefaultBuilder(args)
+            .ConfigureWebHostDefaults(webHostBuilder =>
+            {
+                webHostBuilder.UseStartup<Startup>();
+            })
```
Comment (rynowak) [inline src/Components/Blazor/testassets/HostedInAspNet.Server/Program.cs]: @Tratcher - this is the state of the art right?

**Review by rynowak — APPROVED** [review]
Looks right to me. Nice to see the cleanup and deletions.

File: src/Components/Blazor/testassets/HostedInAspNet.Server/Program.cs (hunk 1 above)
Comment (Tratcher) [reply src/Components/Blazor/testassets/HostedInAspNet.Server/Program.cs]: Almost: https://github.com/aspnet/AspNetCore/blob/96f1c92caa484e6b9c7e5d3768c7dbe70287d87f/src/ProjectTemplates/Web.ProjectTemplates/content/StarterWeb-CSharp/Program.cs#L14-L24

**Comment by davidfowl** [comment]
We need to do this across the board...

**Pull request merged**